Chat/.common.babelrc.js

```diff
@@ -0,0 +1,9 @@
+const plugins = [];
+plugins.push([
+  '@babel/plugin-syntax-typescript',
```

Comment:
nit: is babel just used as a transpiler? If so to reduce config files and to keep the server and app webpacks consistent you can use ts-loader
![image](https://user-images.githubusercontent.com/2684369/149028073-4a2efc9f-f2a8-4a2b-b296-0a05a3ac41d7.png)

Reply:
Oh its webpack already uses that
![image](https://user-images.githubusercontent.com/2684369/149028684-6cc56f5c-ad9f-4d4c-8bf9-0b7a88cd5462.png)

What is babel doing here?

Reply:
It looks like it's maybe not used at all? In which case these babel files and explicit babel dependencies can be removed

Reply:
I was just removing any conditional compilation or local packlet usage. Do we not need this at all? I am trying to take out what we need but still make it easy to merge in any other changes we might need from the UI Lib repo

Reply:
It will be easier to migrate in changes from the UI Library so lets keep this in for now.